src/locale/locales/it/messages.po

```diff
@@ -45,12 +34,12 @@ msgstr ""
 msgid "{0, plural, one {# repost} other {# reposts}}"
 msgstr ""
```

Comment:
```suggestion
msgstr "{0, plural, one {# ripubblicazione} other {# ripubblicazioni}}"
```

I don't speak Italian unfortunately, but looking at the previous translation I think this would be correct?